vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -521,24 +521,35 @@ testNfsServerService() {
 # Tests that the pam.d settings are set correctly, per the function
 # addFailLockDir in <repo-root>/parts/linux/cloud-init/artifacts/cis.sh.
 testPamDSettings() {
+  local os_sku="${1}"
+  local os_version="${2}"
   local test="testPamDSettings"
   local settings_file=/etc/security/faillock.conf
   echo "$test:Start"
 
-  # Existence and format check. The man page https://www.man7.org/linux/man-pages/man5/faillock.conf.5.html
-  # describes the following format for each line:
-  #   Comments start with '#'.
-  #   Blank lines are ignored.
-  #   Lines are of in two forms:
-  #       'setting = value', where settings are lower-case and include '_'
-  #       'setting'
-  #   Whitespace at beginning and end of line, along with around the '=' is ignored.
-  testSettingFileFormat $test $settings_file '^(#|$)' '^[[:space:]]*$' '^[[:space:]]*[a-z_][[:space:]]*' '^[[:space:]]*[a-z_]+[[:space::]]*=[^[:cntrl:]]*$'
-
-  # Look for the setting we specifically set in <repo-root>/parts/linux/cloud-init/artifacts/cis.sh
-  # and ensure it's set to the values we expect.
-  echo "$test: Checking specific settings in $settings_file"
-  testSetting $test $settings_file 'dir' '^[[:space:]]*dir[[:space:]]*=' '=' '/var/log/faillock'
+  # We only want to run this test on Mariner 2.0
+  # So if it's anything else, report that we're skipping the test and bail.
+  if [[ "${os_sku}" != "CBLMariner" || "${os_version}" != "2.0" ]]; then
```

Comment:
am I crazy or are they both listed as CBLMariner in /etc/os-release?

https://github.com/Azure/AgentBaker/blob/166c545f12241d5ee6160d6328c61d672f47e91d/vhdbuilder/release-notes/AKSCBLMarinerV2/gen2/latest.txt#L659-L661

https://github.com/Azure/AgentBaker/blob/166c545f12241d5ee6160d6328c61d672f47e91d/vhdbuilder/release-notes/AKSCBLMariner/gen1/latest.txt#L429-L431

Reply:
oh I get it. skip v1 only but do Ubuntu.

Reply:
Yes, I think they're both `CBL-Mariner`, but then the version check makes sure we're at V2.